### PR TITLE
FixIssue#861: Swipe refresh layout is working on Centerlist,Clientlist,ClientDocumentlist,ClientChargeslist.

### PR DESCRIPTION
**Pull request by ivary43**
- \[x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- \[x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- \[x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by ivary43** [comment]
@tarun0 , @therajanmaurya please review.

**Comment by ivary43** [comment]
@tarun0, @therajanmaurya I have also fixed the screen overlapping issue in Client Charges list (Created when you try to refresh if there is no internet Connection the error screen overlaps with current screen).

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java [hunk 1]
```diff
@@ -151,6 +150,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
             public void onRefresh() {
 
                 mApiRestCounter = 1;
+                if (ll_error.getVisibility() == View.VISIBLE) {
+                    ll_error.setVisibility(View.GONE);
+                    rv_charges.setVisibility(View.VISIBLE);
+                }
+
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientcharge/ClientChargeFragment.java]: Unnecessary space.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientlist/ClientListFragment.java [hunk 2]
```diff
@@ -251,7 +251,11 @@ void onClickCreateNewClient() {
      */
     @Override
     public void onRefresh() {
-        showUserInterface();
+
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientlist/ClientListFragment.java]: Unnecessary new line.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/documentlist/DocumentListFragment.java [hunk 3]
```diff
@@ -146,6 +145,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
 
     @Override
     public void onRefresh() {
+        if (ll_error.getVisibility() == View.VISIBLE) {
+            ll_error.setVisibility(View.GONE);
+        }
+
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/documentlist/DocumentListFragment.java]: Unnecessary new line.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupslist/GroupsListFragment.java [hunk 4]
```diff
@@ -238,27 +238,20 @@ void onClickCreateNewGroup() {
...
-     * groupList.
-     */
-    @OnClick(R.id.noGroupsIcon)
-    public void reloadOnError() {
-        rlError.setVisibility(View.GONE);
-        rv_groups.setVisibility(View.VISIBLE);
-        mGroupsListPresenter.loadGroups(false, 0);
-        mGroupsListPresenter.loadDatabaseGroups();
-    }
+
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupslist/GroupsListFragment.java]: Remove this.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncclientpayloads/SyncClientPayloadsFragment.java [hunk 5]
```diff
@@ -112,6 +111,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             @Override
             public void onRefresh() {
 
+                if (ll_error.getVisibility() == View.VISIBLE) {
+                    ll_error.setVisibility(View.GONE);
+                    rv_payload_clients.setVisibility(View.VISIBLE);
+                }
+
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncclientpayloads/SyncClientPayloadsFragment.java]: Unnecessary new line.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncloanrepaymenttransacition/SyncLoanRepaymentTransactionFragment.java [hunk 6]
```diff
@@ -110,6 +109,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         swipeRefreshLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
             @Override
             public void onRefresh() {
+
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/offline/syncloanrepaymenttransacition/SyncLoanRepaymentTransactionFragment.java]: Unnecessary new line.

**Comment by Aditya-gupta99** [comment]
We are closing this PR as it causes a branch conflict since the java file has now been migrated to kt

**Pull request closed without merging**